.github/workflows/build.yml

```diff
@@ -17,10 +17,13 @@ jobs:
           # in order to calculate the range. i.e. HEAD~20..HEAD includes
           # 20 commits, but including HEAD it needs 21 commits.
           fetch-depth: 21
+      - run: wget https://github.com/google/flatbuffers/releases/download/v22.9.29/Linux.flatc.binary.g++-10.zip
+      - run: unzip Linux.flatc.binary.g++-10.zip
       - run: make install-check-tools
       - run: make check-ltag
       - run: make check-dco
       - run: make check-lint
+      - run: PATH=$PATH:$(pwd) make check-flatc
```

Comment:
Why does cwd have to be in PATH? If it does, that seems like a bug that should be fixed in the Makefile.

Reply:
`cwd` was a mistake on my part. Changed it to `pwd`. The `flatc` binary is installed in `pwd`, since (unlike the Makefile), we are downloading the pre-built binary in the Workflow. (I didn't really want to do `make install-cmake && make install-flatc` on every run.)

Reply:
I see. I might have installed it to /tmp and then added that to PATH. (I really dislike how much we scribble in the source directory during build.) But okay fine.